cime_config/cesm/machines/config_machines.xml

```diff
@@ -468,7 +468,7 @@
     <COMPILERS>intel,gnu,cray</COMPILERS>
     <NODENAME_REGEX>edison</NODENAME_REGEX>
     <MPILIBS>mpt,mpi-serial</MPILIBS>
-    <CESMSCRATCHROOT>$ENV{SCRATCH}</CESMSCRATCHROOT>
+    <CESMSCRATCHROOT>$ENV{CSCRATCH}</CESMSCRATCHROOT>
```

Comment:
Edison now uses same scratch file system as cori, current edison scratch file system scheduled to go off line soon